src/rax.h

```diff
@@ -169,7 +177,7 @@ typedef struct raxIterator {
 
 /* Exported API. */
 rax *raxNew(void);
-rax *raxNewWithMetadata(int metaSize);
+rax *raxNewWithMetadata(int flags, int metaSize);
```

Comment:
nit: I think it would be more consistent with the rest of the source to put the flags param at the end.

Reply:
Done in [ee9d51c](https://github.com/redis/redis/pull/14363/commits/ee9d51cdd5f0eeeadcc483c5bb3c0c289b0f50ee)